### DISCUSSION TITLE
Anyone who wana help me? don’t forward message

### DISCUSSION THREAD

**Discussion by walterpro2024** [aahnik/tgcf#592] [Q&A]
I think I have set up all ,but it doesn't forward any messages.
With ubantu wsl on my windows
I can't find where is the problem.
![image](https://github.com/aahnik/tgcf/assets/166354450/49f3c947-2594-4c4d-bed4-4877f8fbc861)

**Comment by MobCode100** [comment]
> I think I have set up all ,but it doesn't forward any messages.
> With ubantu wsl on my windows
> I can't find where is the problem.
> ![image](https://github.com/aahnik/tgcf/assets/166354450/49f3c947-2594-4c4d-bed4-4877f8fbc861)

Could it be that your source/dest id is not correct?

**Reply by walterpro2024** [reply]
Here is the setup,thank you for reply and help ![image](https://github.com/aahnik/tgcf/assets/166354450/35528629-a3b2-4997-b917-55ae1a8153eb)

**Reply by MobCode100** [reply]
I see on your logs, there is error "/forward". Try restart tgcf.
